matrix-meetings-bot/src/service/MeetingService.ts

```diff
@@ -442,8 +442,12 @@ export class MeetingService {
         ),
       );
     }
-
-    for (const widgetId of room.meeting.widgetIds) {
+    const filteredWidgetIds = room.meeting.widgetIds.filter(
+      (widget) =>
+        widget.startsWith(WidgetType.COCKPIT) &&
+        widget.startsWith(WidgetType.BREAKOUT_SESSIONS),
```

Comment:
This is impossible 🤔. It will never be _both_ starting with `net.nordeck.meetings.widget.cockpit` and `net.nordeck.meetings.widget.breakoutsessions` at the same time.

I would vote to have some unit tests to see what effect this change has (yes, I know that the tests for this class are unpleasant...).

Reply:
It seems like the "Notes" widget in the following example is still updated when I add/remove a widget:

<img width="575" alt="image" src="https://github.com/nordeck/matrix-meetings/assets/720821/307bd19b-9b39-47d7-960b-6d96c61f71fe">